src/proxy-page/proxy-page.service.ts

```diff
@@ -113,12 +117,8 @@ export class ProxyPageService {
     }
     delUnnecessaryCode()(this.context);
     addCustomCss(this.config, style)(this.context);
```

Comment:
Would it make sense to merge `addCustomCss` with `addLibrariesCss` as the two functions are so similar? Or is the intention to keep `addLibrariesCss` dedicated to external css libraries?

Reply:
Yes, I make it intentional to split internal and external libraries but something we can discuss and align later on if we find the split is not relevant.